turbo.json

```diff
@@ -35,6 +35,15 @@
         "playwright-report/**",
         "playwright/.cache/**",
         "test-results/**"
+      ],
+      "env": [
```

Comment:
Minor tweak, but lets change these to `passThroughEnv` ([src](https://turbo.build/repo/docs/crafting-your-repository/using-environment-variables#passthrough-variables)) so changes to them don't potentially kick off fresh/uncached re-builds